docusaurus/docs/migration-guide/v6.x-v7.x/index.md

```diff
@@ -0,0 +1,124 @@
+---
+id: migrate-6.x-to-7.0
+title: 6.x to 7.0
+---
+
+# Migrate plugins from Grafana version 6.x to 7.0
+
+Follow these instructions to upgrade Grafana from version 6.x to 7.0.
+
+## What's new in Grafana 7.0?
+
+Grafana 7.0 introduced a whole new plugin platform based on React. This new platform supersedes the previous Angular-based plugin platform.
+
+Plugins built using Angular still work in the near term, but we strongly encourage new plugin authors to develop with the new platform.
+
+### New data format
+
+Along with the move to React, the new plugin platform introduced a new internal data format called [data frames](../../introduction/data-frames.md).
```

Comment:
I have changed this to [data frames](../introduction/data-frames.md), which is VS Code's suggestion, and it works. Starting with /introduction... doesn't work. Could you put this in a suggestion so I could see the code?

Reply:
But `/introduction...' doesn't work. Do you mean you think it would be better to use `https://grafana.github.io/plugin-tools/docs/introduction/data-frames`? If so, I wouldn't want to do this given that we are planning to publish this on the dev portal, which I presume will require relative links.

Reply:
Changing this line to `Along with the move to React, the new plugin platform introduced a new internal data format called [data frames](/introduction/data-frames.md)` builds in dev and production. Can you expand a bit on what doesn't work? It should resolve to the same place in the browser.

Reply:
The link doesn't resolve for me, and VS code won't suggest it. I'll see if I can take a screen video capture, but I'm not well versed in the Snag-It screen capture tool yet. Or, we can get on a short call next week and you can see what I'm talking about.

Reply:
Let's forget this for now and move on. It's not that big an issue and I'd rather unblock you.